src/clarity-angular/data/datagrid/_datagrid.clarity.scss

```diff
@@ -785,6 +789,75 @@
         }
     }
 
+    .datagrid-compact {
+        .datagrid-column .datagrid-column-separator {
+            &::after {
+                height: calc(100% + 0.5 * #{$clr-datagrid-column-separator-expandby} - #{$clr-default-borderwidth});
+                top: calc(-0.25 * #{$clr-datagrid-column-separator-expandby} + 1px);
+            }
+            .datagrid-column-handle-tracker {
+                top: 0.5*$clr-datagrid-column-handle-tracker-top;
+            }
+        }
+        .datagrid-cell {
+            clr-icon {
```

Comment:
Same comment as above regarding the `clr-icon` selector.

Reply:
This one is targeting all icons inside `.datagrid-cell`.